guvnor-bom/guvnor-bom.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@desmax74 there's a [general recommendation](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems) to add `*.iml` files to .gitignore. 

Is there any reason to include this particular iml file into git?

Reply:
Isn't a manual addition, let me check why this is here

Reply:
Found the reason, going to delete